spec/fixtures/files/test.csv

```diff
@@ -1,7 +1,7 @@
-UID;Name;Start date;End date;City;Country;Region;Website;Notes
-2017001;Deccan RubyConf 2017;Aug 12, 17;Aug 12, 17;Gent;Belgium;Europe;http://www.deccanrubyconf.org/;
-2017002;JSFoo 2017;Sep 15, 17;Sep 16, 17;Bangalore;India;Asia Pacific;https://jsfoo.in/2017/;
-2017003;;;;Delhi;India;Asia Pacific;https://in.pycon.org/;1st week of Nov (dates are to be defined)
-2017004;GHC India 2017;17/11/2017;15/11/2017;Bangalore;India;Asia Pacific;https://ghcindia.anitaborg.org/;They have student scholarship applications. Deadline: June 30, 2017
-2018005;PyCon Pune 2018;;;Pune;India;Asia Pacific;https://pune.pycon.org/;Dates are to be defined
-2017006;CheesyRuby;07/07/2017;15/07/2017;Amsterdam;NL;Europe;www.example.com;Duly Noted
+UID,Name,Start date,End date,City,Country,Region,Website,Notes
+2017001,Deccan RubyConf 2017,"Aug 12, 17","Aug 12, 17",Gent,Belgium,Europe,http://www.deccanrubyconf.org/,
+2017002,JSFoo 2017,"Sep 15, 17","Sep 16, 17",Bangalore,India,Asia Pacific,https://jsfoo.in/2017/,
+2017003,,,,Delhi,India,Asia Pacific,https://in.pycon.org/,1st week of Nov (dates are to be defined)
+2017004,GHC India 2017,"17/11/2017","15/11/2017",Bangalore,India,Asia Pacific,https://ghcindia.anitaborg.org/,They have student scholarship applications. Deadline: June 30, 2017
```

Comment:
Let us use a real file exported by Google please. As you stated before, CSV is a very loose format. We can't accomodate any CSV format under the sky. Instead we should aim for what Google exports right now (since one cannot customise the output format) and since Google is what Maria is using right now.